.changeset/wide-wombats-own.md

```diff
@@ -0,0 +1,8 @@
+---
+"@workflow/web": patch
+"@workflow/world-embedded": patch
+---
+
+Web: Allow filtering by workflow name and status on the runs list view
+
+World-embedded: filter by workflowName/status if passed
```

Comment:
These should be 2 separate changelog files. No need to use the prefix then since it autoamtically gets associated with the right package